src/main/java/com/redhat/devtools/lsp4ij/client/features/EditorBehaviorFeature.java

```diff
@@ -99,6 +99,17 @@ public boolean isEnableTextMateNestedBracesImprovements(@NotNull PsiFile file) {
         return false;
     }
 
+    /**
+     * Whether or not the custom file view provider for TextMate files is enabled.
+     *
+     * @param file the file
+     * @return true if the custom file view provider for TextMate files is enabled; otherwise false
+     */
+    public boolean isEnableTextMateFileViewProvider(@NotNull PsiFile file) {
```

Comment:
Feature flag defaults to `false`/disabled for non-user-defined language server definitions.